src/aleph/http/server.clj

```diff
@@ -168,13 +168,13 @@
               ;; handle request on a separate thread
               (try
                 (d/future-with executor
-                  (handler req'))
+                               (handler req'))
```

Comment:
NB: as a body macro, this should be indented only 2 spaces, for future reference